pyproject.toml

```diff
@@ -25,7 +25,7 @@ ignore = ["E501", "E741", "E731", "F401"]  # Easy ignore for getting it running
 
 [tool.poetry]
 name = "scale-nucleus"
-version = "0.16.7"
+version = "0.16.9"
```

Comment:
Nitpick, as I see, version `0.16.8` has not been released so maybe it would make sense to combine these changes into that version instead of creating a new one?